src/libopensc/card-cardos.c

```diff
@@ -895,8 +1088,21 @@ cardos_compute_signature(sc_card_t *card, const u8 *data, size_t datalen,
 
 	assert(card != NULL && data != NULL && out != NULL);
 	ctx = card->ctx;
+	priv = (cardos_data_t*)card->drv_data;
```

Comment:
clang-format suggested the following change:


```
-	priv = (cardos_data_t*)card->drv_data;
+	priv = (cardos_data_t *)card->drv_data;
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->